### PR TITLE
P0 - Input

### PR DESCRIPTION
**Pull request by rishmi5h**
Created
- Inputs
 preview link : https://deploy-preview-4--sigma-ui.netlify.app/pages/docs.html#input

Screenshot 2022-02-10 at 9 04 00 PM

Screenshot 2022-02-10 at 9 04 17 PM

validation

Screenshot 2022-02-10 at 9 04 34 PM

### CODE REVIEW

File: pages/docs.html [hunk 1]
```diff
@@ -448,8 +448,85 @@ <h3 class="m-b-2">Example of Diferent Alerts</h3>
             </iframe>
           </div>
         </div>
+        <div id="input" class="m-t-4">
```
Comment (zabihshaik) [inline pages/docs.html:451]: Better to use mt-4 instead of m-t-4

**Review by zabihshaik — COMMENTED** [review]
Everything looks good. Just a single comment. Please check.

File: pages/docs.html (hunk 1 above)
Comment (rishmi5h) [reply pages/docs.html:451]: I have defined padding and margin like m-t-4, p-tb-4, here p-tb-4 is more understandable I guess as compared to ptb-4

**Review by sonishreyas — APPROVED** [review]
Looks good to me

**Pull request merged**